e2e/nextjs/instrumentation.ts

```diff
@@ -1,10 +1,20 @@
 // instrumentation.ts
 import CONSTANTS from '@/app/constants'
-import { registerHighlight } from '@highlight-run/next'
+// import { registerHighlight } from '@highlight-run/next'
```

Comment:
should we remove the import?

Reply:
Yeah. I was of a split mind on this, but you're pulling me over to removing it.